File: Python/Problem-7.py
# Write a Program to Check The Given Number is Even Or ODD
number = int(input("Enter Your Number: "))
if number < 0:
  print(f"{number} is nagitive Plese Enter only Positive Number")
elif number % 2 == 0:
  print(f"{number} is Even")
else:
  print(f"{number} is ODD")

File: JavaScript/Problem-7.js
// # Write a Program to Check The Given Number is Even Or ODD
const prompt = require("prompt-sync")();

var number = parseInt(prompt("Enter Any Number: "));
if (number <= 0) {
  console.log("Please enter Positive Number");
} else if (number % 2 == 0) {
  console.log(`${number} is Even`);
} else {
  console.log(`${number} is ODD`);
}
